src/prompts/text.rs

```diff
@@ -136,6 +142,12 @@ impl<'a> Text<'a> {
         self
     }
 
+    /// Sets the base input.
```

Comment:
``````suggestion
    /// Sets the initial value of the prompt's text input.
    ///
    /// If you want to set a default value for the prompt, returned when the user's submission is empty, see [with_default].
```